units-databinding/src/main/kotlin/dev.icerock.moko.units.databinding/BindingAdapters.kt

```diff
@@ -95,3 +99,40 @@ private fun getLifecycleOwnerFromContext(context: Context): LifecycleOwner {
     if (context is ContextWrapper) return getLifecycleOwnerFromContext(context.baseContext)
     throw IllegalArgumentException("context $context not implement LifecycleOwner")
 }
+
+fun RecyclerView.bindUnits(lifecycleOwner: LifecycleOwner, units: LiveData<List<UnitItem>>) {
```

Comment:
it's not databinding adapter. it's seems like it for viewbinding

Reply:
moved to units-viewbinding/.../RecyclerViewBinding.kt